Holodeck/Holodeck.py

```diff
@@ -34,6 +37,20 @@ def _get_worlds_map():
     return holodeck_worlds
 
 
+# Resolves python 2 issue with json loading in unicode instead of string
+def _convert_unicode(value):
+    if isinstance(value, dict):
+        return {_convert_unicode(key): _convert_unicode(value)
+                for key, value in value.iteritems()}
+    elif isinstance(value, list):
+            return [_convert_unicode(item) for item in value]
+    elif isinstance(value, unicode):
+        new_val = value.encode('utf-8')
+        return value.encode('utf-8')
```

Comment:
Why is value.encode called twice?

Reply:
Woops. Looks like that was left over from a different change. I'll fix that.